src/modules/75-ce/components/COGatewayConfig/steps/ResourceFulfilment.tsx

```diff
@@ -92,7 +92,7 @@ const ResourceFulfilment: React.FC<ResourceFulfilmentProps> = props => {
     }
   }
 
-  const handleSpotChange = (formik: FormikContext<any>, val: string) => {
+  const handleSpotChange = (formik: FormikContextType<any>, val: string) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232251)
```suggestion
  const handleSpotChange = (formik: FormikContextType<unknown>, val: string) => {
```